### ISSUE TITLE
Impossible to clone submodule

### ISSUE THREAD

**Issue by thomasfla** [Gepetto/gepetto-viewer#189]
Hi, it seems that the submodule can not be clone due to a GitHub authentication error.
```
tflayols@kiso:~/git$ git clone git@github.com:Gepetto/gepetto-viewer.git --recursive 
Clonage dans 'gepetto-viewer'...
remote: Enumerating objects: 4354, done.
remote: Counting objects: 100% (393/393), done.
remote: Compressing objects: 100% (294/294), done.
remote: Total 4354 (delta 166), reused 206 (delta 86), pack-reused 3961
Réception d'objets: 100% (4354/4354), 2.35 Mio | 1.11 Mio/s, fait.
Résolution des deltas: 100% (2692/2692), fait.
Sous-module 'cmake' (git://github.com/jrl-umi3218/jrl-cmakemodules.git) enregistré pour le chemin 'cmake'
Clonage dans '/home/tflayols/git/gepetto-viewer/cmake'...
fatal: erreur distante : 
  The unauthenticated git protocol on port 9418 is no longer supported.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
fatal: le clonage de 'git://github.com/jrl-umi3218/jrl-cmakemodules.git' dans le chemin de sous-module '/home/tflayols/git/gepetto-viewer/cmake' a échoué
Impossible de cloner 'cmake'. Réessai prévu
Clonage dans '/home/tflayols/git/gepetto-viewer/cmake'...
fatal: erreur distante : 
  The unauthenticated git protocol on port 9418 is no longer supported.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
fatal: le clonage de 'git://github.com/jrl-umi3218/jrl-cmakemodules.git' dans le chemin de sous-module '/home/tflayols/git/gepetto-viewer/cmake' a échoué
Impossible de cloner 'cmake' pour la seconde fois, abandon
```

**Comment by florent-lamiraux** [comment]
On which version are you ? I think @nim65s fixed the issue in the devel branch.
Otherwise, you can type
```git config --global url."https://".insteadOf git://```
and try again.

**Comment by nim65s** [comment]
Yes, this is fixed in #180, and the release of that is in progress in #188. Sorry for the inconvenience

**Comment by thomasfla** [comment]
Thanks, I ended up installing it from robotpkg.
Maybe the binary installation could be added in the doc?
